README.md

```diff
@@ -47,11 +47,17 @@ This project is helpful if you have both [`ecto`](https://github.com/elixir-lang
 
 ### Custom type
 
+Append defining block before the schema module.
+
 ```elixir
-defmodule MyProject.Message.Status do
+defmodule MyProject.Messages.Message.Status do
```

Comment:
Why such change?

Reply:
Phoenix 1.3 is using Contexts as an internal APIs.

Reply:
This library is not related to Phoenix anyhow.